### DISCUSSION TITLE
Welcome to SPX-GC Discussions!

### DISCUSSION THREAD

**Discussion by TuomoKu** [TuomoKu/SPX-GC#16] [General]
## 👋 Hey, welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Share your SPX-GC projects, ideas, setup or hack to inspire others.

**Comment by hamrejzak** [comment]
Hello, 

great project ad work! 

I have a question regarding special function as you have set up function **stopAll** in demoFunctions.js 

How to set up a function, which will play all CG in rundown? 

Thank you very much for answer! 

Sincerely, 

Petr H.

**Reply by TuomoKu** [reply]
Hey, that's pretty easy 😉

There is a built in function _playItem(DomElement, 'play')_ which will just play an item on the rundown.

So you can copy the **stopAll()** function as **playAll()** and replace
`continueUpdateStop('stop', templateItem);`
with 
`playItem(templateItem,'play')`

and it will **play** an item instead of **stopping** it. Then you will need to add that button to your project or globally in app config. I take it you know how to do that already?

As you can imagine this can get quite humorous on most rundowns, but hey, if you need one you need one 👍

Hope this helps

--Tuomo

**Reply by hamrejzak** [reply]
Hey, 

thank you very much for really quick response! Right now, it works! Adding button was the easy way, but I wosn´t sure about the exact function. 

Sincerely, 

Petr H.
